lib/Image.php

```diff
@@ -108,55 +108,13 @@ public function __toString() {
 
 	/**
 	 * Get a PHP array with pathinfo() info from the file
+	 * @deprecated since 2.0 Functionality will no longer be supported in future releases.
 	 * @return array
 	 */
 	public function get_pathinfo() {
 		return pathinfo($this->file);
```

Comment:
Wouldn’t this need a call to `Helper::deprecated()`?

Reply:
Yep! will address in the next commit